packages/cdktf-cli/bin/cmds/ui/models/terraform-cloud.ts

```diff
@@ -320,4 +343,26 @@ export class TerraformCloud implements Terraform {
     await Promise.race([ready(), timeout()]);
     logger.debug('Configuration Version is ready in Terraform Cloud');
   }
+
+  public outputParser(line: string): DeployingResource | undefined {
+    let message: CloudLog;
+    try {
+      message = JSON.parse(line) as CloudLog
+    } catch (error) {
+      throw new Error(`Could not parse Terraform Cloud output: ${line}`);
```

Comment:
We might want to just print or log the line rather than throw.

Reply:
Good point, I will change it 👍